test/python/transpiler/test_calibrationbuilder.py

```diff
@@ -266,7 +266,8 @@ def build_reverse(
     @data(-np.pi / 4, 0.1, np.pi / 4, np.pi / 2, np.pi)
     def test_rzx_calibration_cr_pulse_stretch(self, theta: float):
         """Test that cross resonance pulse durations are computed correctly."""
-        backend = Fake27QPulseV1()
+        with self.assertWarns(DeprecationWarning):
+            backend = Fake27QPulseV1()
         inst_map = backend.defaults().instruction_schedule_map
```

Comment:
This can be left for a follow-up, but if I'm not mistaken these tests should be migrated to use `GenericBackendV2`

Reply:
They use `defaults()` tho

Reply:
The `instruction_schedule_map` can be retrieved from the target in the V2 model as `backend.target.instruction_schedule_map()`, no need to go through `defaults`